frontend/src/Editor/Inspector/Elements/Code.jsx

```diff
@@ -25,6 +38,7 @@ export const Code = ({ param, definition, onChange, paramType, componentMeta, cu
         lineWrapping={true}
         className={options.className}
         onChange={(value) => handleCodeChanged(value)}
+        componentName={`widget/${componentName}::${getfieldName}`}
```

Comment:
I think you are already appending `widget/**/::` using `getPopoverHeader` and this will cause it to repeat?

Reply:
`getPopoverHeader ` we only get the table's col fields name. probably should name it correctly.
and `Code.jsx` is only referred  in `renderElement` function and `renderElement` is referred in `table chart and inspector .jsx`! that's why hardcoded "widget/..."